contributions/api/controllers/contributions.py

```diff
@@ -883,7 +889,8 @@ def admin_reviewers_post(token_info):
     logging.info("Contribution Admin POST " + json.dumps(msg_json))
 
     reviewer_id = str(dataset['_id'])
-
+    # send email when new reviewer is added
+    send_email_new_reviewer(dataset['githubUsername'])
     return rs_handlers.return_id(msg, 'id', reviewer_id)
```

Comment:
This one as well. Maybe making it to send just one mail to everyone, instead of several emails to single person unless we need to hide if others email

Reply:
send_email_new_reviewer() sends an email notification to the newly added reviewer. This method is aimed at notifying a user if the user has been added as a reviewer. It might be unnecessary to notify all reviewers of a newly added person.